pkgs/development/python-modules/distrax/default.nix

```diff
@@ -23,6 +23,14 @@ buildPythonPackage rec {
     hash = "sha256-A1aCL/I89Blg9sNmIWQru4QJteUTN6+bhgrEJPmCrM0=";
   };
 
+  # AttributeError: jax.core.Literal was removed in JAX v0.6.0. Use jax.extend.core.Literal instead,
+  # and see https://docs.jax.dev/en/latest/jax.extend.html for details.
```

Comment:
I propose that we create or find issues on these projects letting them know about these issues and then link them here. Not strictly necessary, but seems like the right thing to do.